.github/workflows/release-template-comment.yml

```
@@ -0,0 +1,12 @@
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+
+jobs:
+  call-release-workflow:
```

```suggestion
  call-release-checklist-workflow:
```
worth distinguishing this workflow from the release workflow.

---

That's due to the example in ASFHyP3/actions; fixed here:
https://github.com/ASFHyP3/actions/pull/67